Sources/xcstrings-tool/Generate.swift

```diff
@@ -43,6 +43,12 @@ struct Generate: ParsableCommand {
         help: "Modify the Access Control for the generated source code"
     )
     var accessLevel: AccessLevel?
+    
+    @Option(
+        name: .customShort("s"),
+        help: "Treat underscores the same as other separators when converting keys to camel case."
+    )
+    var convertFromSnakeCase: Bool?
```

Comment:
Just one thing here.. I think that this should be a `@Flag` (like `verbose`) rather than an `@Option`. iirc, options require something like `--convert-from-snake-case true` whereas the flag itself is a boolean and omitting it is considered `false`

```suggestion
    @Flag(
        name: .long,
        help: "Treat underscores the same as other separators when converting keys to camel case."
    )
    var convertFromSnakeCase: Bool = false
```

Secondly, since `convertFromSnakeCase` isn't really a common option, the `-s` short name might be seen as a bit confusing. I think in this case it would be better to only support the full name by specifying `.long` as the naming style.

Reply:
Using a flag also makes this non-optional so you would have to remove the `?? false` in the `Configuration` init 🙏

Reply:
All good points, lets go with a flag and default value 👍🏻